.coveragerc

```diff
@@ -11,6 +11,9 @@ omit =
     homeassistant/components/abode.py
     homeassistant/components/*/abode.py
 
+    homeassistant/components/ads.py
```

Comment:
You need to update the path to the package here. Sorry that I missed that.

Reply:
Damn, I forgot about this. Should work now.